compressor/storage.py

```diff
@@ -61,7 +61,9 @@ def save(self, filename, content):
 
 
 compressor_file_storage = SimpleLazyObject(
-    lambda: get_storage()
+    lambda: storages.create_storage(
+        {"BACKEND": "compressor.storage.CompressorFileStorage"}
+    )
```

Comment:
This is the actual fix. This **should not** be affected by user settings. (This was the error in v4.5)